vissl/config/defaults.yaml

```diff
@@ -195,7 +195,7 @@ config:
       # no valid values other than 0 or None), unclear if beneficial. Default =
       # None.
       # TRANSFORMS:
-      #   - name: VisslRandAugment
+      #   - name: RandAugment
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.